README.md

```diff
@@ -194,6 +194,51 @@ E.g. `Foo = [name: tstr, age: uint]` is equivalent to `Foo = [tstr, uint]`.
 
 See [test3_simple](tests/decode/test3_simple/) for CDDL example code.
 
+Introduction to CBOR
+====================
+
+CBOR's format is described well on Wikipedia
+ - https://en.wikipedia.org/wiki/CBOR
+but here's a synopsis:
+
+Encoded CBOR data elements look like this.
+
+```
+| Header             | Value                  | Payload                   |
+| 1 byte             | 0, 1, 2, 4, or 8 bytes | 0 - 2^64-1 bytes/elements |
+| 3 bits | 5 bits    |
+| Major  | Additional|
+| Type   | info      |
+```
+
+The available major types can be seen in zcbor_major_type_t.
+
+For all major types, Values 0-23 are encoded directly in the "Additional info", meaning that the "Value" field is 0 bytes long.
```

Comment:
```suggestion
For all major types, Values 0-23 are encoded directly in the _Additional info_, meaning that the _Value_ field is 0 bytes long.
```
nit: What do you think of italics for referencing sections instead of quotations? For example:

> For all major types, Values 0-23 are encoded directly in the _Additional info_, meaning that the _Value_ field is 0 bytes long.

Same for other instances in the file.

Reply:
Good idea, added italics to all instances of Header, Value, Payload, and Additional info.